lib/wallaby/session.ex

```diff
@@ -153,6 +153,15 @@ defmodule Wallaby.Session do
   end
 
   @doc """
+  Gets the title for the current page
+  """
+  @spec title(t) :: String.t
+
+  def title(session) do
```

Comment:
I would suggest making the function name more specific (e.g. `get_page_title()`) to follow the naming of similar functions `get_window_size()`, `get_current_path()` and to indicate this does not operate on DOM nodes.